### DISCUSSION TITLE
Welcome to medly-components Discussions!

### DISCUSSION THREAD

**Discussion by gmukul01** [medly/medly-components#215] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bhusnar5044** [comment]
Hi @gmukul01,
I've faced some issues with a library in the next.js environment.
because of the SSR feature in nextJs it's giving **window object not defined** such errors
we can avoid such errors by using window object in the useEffect method or using it conditionally.

**Reply by gmukul01** [reply]
@Bhusnar5044 Thanks for reporting this issue brother. I have created a card in backlog to fix this and soon we will fix that.

**Reply by gmukul01** [reply]
@Bhusnar5044  Thank you so much for raising the PR #411 .
